src/dashboard/src/components/helpers.py

```diff
@@ -304,14 +304,24 @@ def processing_config_path():
     )
 
 
-def stream_file_from_storage_service(url, error_message='Remote URL returned {}'):
-    stream = requests.get(url, stream=True, timeout=django_settings.STORAGE_SERVICE_CLIENT_TIMEOUT)
+def stream_file_from_storage_service(url,
+                                     error_message='Remote URL returned {}',
+                                     preview_file=False):
+    # Repetetive or constant values to use below when seeting the headers.
+    header_content_disposition = 'Content-Disposition'
+    content_disposition_inline = 'inline'
+    storage_timeout = django_settings.STORAGE_SERVICE_CLIENT_TIMEOUT
+    stream = requests.get(url,
+                          stream=True,
+                          timeout=storage_timeout)
     if stream.status_code == 200:
         content_type = stream.headers.get('content-type', 'text/plain')
         content_disposition = stream.headers.get('content-disposition')
         response = StreamingHttpResponse(stream, content_type=content_type)
-        if content_disposition:
-            response['Content-Disposition'] = content_disposition
+        if preview_file:
+            response[header_content_disposition] = content_disposition_inline
+        elif content_disposition:
+            response[header_content_disposition] = content_disposition
```

Comment:
Nitpick: I think the style here is rather verbose. There's a lot of this in Archivematica but you can also find more recent code that I find more concise that feels more Pythonic, probably introduced by Holly and Joel. E.g. you could go from 6 to 2 lines of code with:

    response['Content-Disposition'] = \ 
        'inline' if preview_file else content_disposition

I believe that this is ultimately up to you. But you're going to see other places where a lot is going on inline, e.g.:

https://github.com/artefactual/archivematica/blob/4e84fa7bf4392b8cecf7bce8abb6009ed71d50ea/src/archivematicaCommon/lib/bindpid.py#L307-L357

Reply:
I think for an `if` the compound style goes somewhat against the readability of Python, we chatted before and PEP8 seems to be a bit in between on it. http://legacy.python.org/dev/peps/pep-0008/#other-recommendations I will try and keep it in mind where I can become more compact elsewhere for sure.